src/components/Comments/WriteComment.js

```diff
@@ -72,11 +75,16 @@ const WriteComment = ({
       }
     });
   }, []);
+  React.useEffect(() => {
+    if (isKeyboardOpen) {
+      commentInputRef.current.focus();
+    }
+  }, [isKeyboardOpen, commentInputRef.current]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This useEffect hook is used to focus on the comment input when the keyboard is open. This seems like a good addition for user experience, but make sure that this doesn't interfere with other interactions in your application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->